### PR TITLE
update tasks samples to support new api version

### PR DESCRIPTION
**Pull request by bart-at-google**

### CODE REVIEW

File: samples/server.js [hunk 1]
```diff
@@ -33,8 +33,9 @@ app.get('/', (req, res) => {
 
 app.post('/log_payload', (req, res) => {
   // Log the request payload
-  console.log('Received task with payload: %s', req.body);
-  res.send(`Printed task payload: ${req.body}`).end();
+  const body = String(req.body);
```
Comment (JustinBeckwith) [inline samples/server.js]: You shouldn't need to do the explicit cast here

File: samples/server.js [hunk 2]
```diff
@@ -33,8 +33,9 @@ app.get('/', (req, res) => {
...
+  console.log('Received task with payload: %s', body);
```
Comment (JustinBeckwith) [inline samples/server.js]: You can use the same string interpolation syntax as below:
```js
console.log(`Received task with payload: ${body}`);
```

File: samples/server.js [hunk 3]
```diff
@@ -33,8 +33,9 @@ app.get('/', (req, res) => {
...
+  res.send(`Printed task payload: ${body}`).end();
```
Comment (JustinBeckwith) [inline samples/server.js]: You can just call `res.end('text....');` instead of using `send` and `end` :)

File: samples/server.js (hunk 1 above)
Comment (bart-at-google) [reply samples/server.js]: Hmmm, you're right. I must have incorrectly attributed a different problem to this. I'm going to revert all of my changes in this file.

**Pull request merged**